aix/SPECS/4.2.0/wazuh-agent-4.2.0-aix.spec

```diff
@@ -101,11 +101,23 @@ if [ $1 = 1 ]; then
 fi
 
 if [ $1 = 2 ]; then
-  touch %{_localstatedir}/tmp/wazuh.restart
-  /etc/rc.d/init.d/wazuh-agent stop > /dev/null 2>&1 || :
+  if %{_localstatedir}/bin/wazuh-control status 2>/dev/null | grep "is running" > /dev/null 2>&1; then
+    /etc/rc.d/init.d/wazuh-agent stop > /dev/null 2>&1 || :
+    touch %{_localstatedir}/tmp/wazuh.restart
+  fi
 fi
 
 %post
+if [ $1 = 2 ]; then
+  if [ -d %{_localstatedir}/logs/ossec ]; then
+    cp -rT %{_localstatedir}/logs/ossec/ %{_localstatedir}/logs/wazuh
```

Comment:
This needs to be tested. https://www.ibm.com/support/knowledgecenter/ssw_aix_72/c_commands/cp.html
Flag `T` doesn't appear in the AIX cp command guide. Request AIX 6 if needed.

Reply:
Done in 38e5863